docs/Usage_example_HealthRI_core.ipynb

```diff
@@ -0,0 +1,441 @@
+      "|    1 | Gryffindor research project | Impact of muggle technical inventions on word's magic presense       | Hermione Granger | https://harrypotter.fandom.com/wiki/Hermione_Granger | magic, technic, muggles             |\n",
+      "|    2 | Slytherin research project  | Comarative analysis of magic powers of muggle-born and blood wizards | Draco Malfoy     | https://harrypotter.fandom.com/wiki/Draco_Malfoy     | half-blood, muggle-born, pure-blood |\n",
+      "|    3 | Ravenclaw research project  | Linguistic analysis of spells                                        | Luna Lovegood    | https://harrypotter.fandom.com/wiki/Luna_Lovegood    | linguastics spell                   |\n",
+      "|    4 | Hufflepuff research project | Domestics elfs reproduction rate changes in the last 100 years       | Dora Williams    | https://harrypotter.fandom.com/wiki/Dora_Williams    | domestic elf                        |\n",
+      "+------+-----------------------------+----------------------------------------------------------------------+------------------+------------------------------------------------------+-------------------------------------+\n"
+     ]
+    }
+   ],
+   "source": [
+    "from tabulate import tabulate\n",
+    "import pandas as pd\n",
+    "\n",
+    "df = pd.read_csv(\"./example_data.csv\", sep=\";\")\n",
+    "print(tabulate(df, headers='keys', tablefmt='psql', showindex=False))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": []
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "A few fields already exist! We try to map all mandatory fields now based on what we have."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# We split keywords by the comma, and put them in a List\n",
+    "df[\"keywords\"] = df[\"keywords\"].apply(lambda x: x.split(\",\"))\n",
+    "df.rename(columns={\"keywords\": \"keyword\"}, inplace=True)\n",
+    "\n",
+    "# We rename \"id\" to identifier\n",
+    "# Note: cardinality in DCAT-AP is 0..* but in Health-RI is 1..1\n",
+    "# So it should NOT be in a list\n",
+    "df[\"id\"] = df[\"id\"].apply(lambda x: str(x))\n",
+    "df.rename(columns={\"id\": \"identifier\"}, inplace=True)\n",
+    "\n",
+    "# Description can stay as is, but we put it in a List (reason: i18n support).\n",
+    "df[\"description\"] = df[\"description\"].apply(lambda x: [x])\n",
+    "\n",
+    "# Name is called \"title\" in dcat. We first put it in a List, then rename it\n",
+    "df[\"name\"] = df[\"name\"].apply(lambda x: [x])\n",
+    "df.rename(columns={\"name\": \"title\"}, inplace=True)\n",
+    "\n",
+    "# creator is a foaf:Agent. We add the mandatory tags for it\n",
+    "df[\"creator\"] = df.apply(lambda x: [{'name': [\"author_name\"], x[\"author_id\"]], axis=1)\n",
```

Comment:
`SyntaxError: closing parenthesis ']' does not match opening parenthesis '{'`. I'm not too familiar with pandas, so purely on syntax should it be sth like 

```suggestion
    "df[\"creator\"] = df.apply(lambda x: [{'name': x[\"author_name\"]}, 'identifier': x[\"author_id\"]], axis=1)\n",
```
❓

Reply:
Thanks for spotting this!